rotkehlchen/externalapis/cowswap.py

```diff
@@ -92,7 +92,7 @@ def get_order_data(self, order_uid: str) -> tuple[int, str]:
                 raw_fee_amount = int(data['executedFeeAmount'])
                 order_type = data['class']
             else:
-                raw_fee_amount = int(data.get('executedSurplusFee', 0))
+                raw_fee_amount = int(data.get('executedSurplusFee', data.get('executedFee', 0)))
```

Comment:
what's this and why this change now?

Reply:
I redecoded the cowswap tests and noticed that the API response has changed regarding what we have. `executedSurplusFee` is not always there and `executedFee` is always present. We were missing some fees in cowswap